doc/man1/openssl-req.pod.in

```diff
@@ -324,11 +326,27 @@ Sets the "CA" private key to sign a certificate with.
 The private key must match the public key of the certificate given with B<-CA>.
 If this option is not provided then the key must be present in the B<-CA> input.
 
+=item B<-startdate> I<date>
+
+This allows the start date to be explicitly set. The format of the
+date is YYMMDDHHMMSSZ (the same as an ASN1 UTCTime structure), or
+YYYYMMDDHHMMSSZ (the same as an ASN1 GeneralizedTime structure). In
+both formats, seconds SS and timezone Z must be present.
```

Comment:
Look like it can also be "today"

Reply:
Correct! Maybe this needs to be documented also for `ca` app?

Reply:
After double-checking this: `today` is only used internally.
The options only accept dates in ASN1 format.

Reply:
No, all uses of `set_cert_times()` support `startdate` (and thus `-not_before`) being "today".
BTW, you could easily extend this also to `enddate` (for use with `-not_after`).